sensors/lego_sensor_class.c

```diff
@@ -67,7 +67,8 @@
  * .    - `u16`: Unsigned 16-bit integer (ushort)
  * .    - `s16`: Signed 16-bit integer (short)
  * .    - `s16_be`: Signed 16-bit integer, big endian
- * .    - `s32`: Signed 32-bit integer (int)
+ * .	- `s32`: Signed 32-bit integer (int)
+ * .	- `s32_be`: Signed 32-bit integer, big endian
```

Comment:
It looks like you are still using tabs here. This is actually parsed as markdown for our website, so it is important to have 4 spaces here.